packages/istanbul-report-modern-html/assets/modern.css

```diff
@@ -0,0 +1,257 @@
+body,
+html {
+    margin: 0;
+    padding: 0;
+    height: 100%;
+}
+body {
+    font-family: Helvetica Neue, Helvetica, Arial;
+    font-size: 14px;
+    color: #333;
+}
+.small {
+    font-size: 12px;
+}
+*,
+*:after,
+*:before {
+    -webkit-box-sizing: border-box;
+    -moz-box-sizing: border-box;
+    box-sizing: border-box;
+}
+h1 {
+    font-size: 20px;
+    margin: 0;
+}
+a {
+    color: #0074d9;
+    text-decoration: none;
+}
+a:hover {
+    text-decoration: underline;
+}
+.strong {
+    font-weight: bold;
+}
+.space-top1 {
+    padding: 10px 0 0 0;
+}
+.pad1y {
```

Comment:
This suggestion is probably overkill, but, I like having a consistent framework for things like this (not that you have made inconsistencies here!) -- I'm a huge fan of https://styled-system.com/ 

What do you think?

Reply:
This css file is inherited from the existing report - I would have tried styled components if I was starting from scratch, but actually, I took the existing css, removed unused classes and added a small amount of css at the end.
Someone would be welcome to refactor this and improve upon the styling in a later PR imo.